appveyor.yml

```diff
@@ -26,7 +26,7 @@ build:
   
 after_build:
  - ps: | # Package artifact
-    7z a -m0=LZMA2 -mx9 $env:BUILD .\build-windows\src\emulator\Release\*
```

Comment:
Is this OK? Do we need to do this to change the icon?

Reply:
no, it is for move executable on bin folder, same in my scree, obligatotory for compile fine on appveyor